x-pack/plugins/secops/public/store/local/timeline/helpers.ts

```diff
@@ -251,6 +256,108 @@ const addProviderToTimeline = (
     },
   };
 };
+
+interface AddTimelineColumnParams {
+  id: string;
+  column: ColumnHeader;
+  timelineById: TimelineById;
+}
+
+export const addTimelineColumn = ({
+  id,
+  column,
+  timelineById,
+}: AddTimelineColumnParams): TimelineById => {
+  const timeline = timelineById[id];
+
+  const alreadyExistsAtIndex = timeline.columns.findIndex(c => c.id === column.id);
```

Comment:
I would use a `find` rather than `findIndex`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Since you are using it as a boolean expression below.

Reply:
Would you mind taking another look at this? It's actually used as an index throughout the function, which means the "truthyness" alone is not sufficient (and therefore `some` would not be applicable here.)

Reply:
Sure, looks good.

Reply:
> Would you mind taking another look at this? It's actually used as an index throughout the function, which means the "truthyness" alone is not sufficient (and therefore some would not be applicable here.)

That's the pattern in the rest of the file, but you were correct about the usage here: it's only being used as a boolean because it's appending to the end. Apologies for the confusion, and I implemented your suggestion, thanks! 😄